eu.fortysixandtwo.chessclock.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id" : "eu.fortysixandtwo.chessclock",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "chess-clock",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+	{
+            "name": "gsound",
```

Comment:
Indent

Reply:
Sorry, how exactly is this supposed to look like?

If I (visually) compare against e.g.
https://github.com/flathub/io.github.waylyrics.Waylyrics/blob/master/io.github.waylyrics.Waylyrics.json
I don't see a difference (?)

Reply:
two wrongs don't make a right.

Reply:
Sure, but your post also does not help me understanding **how** exactly it's wrongly indented
or what would be **right**.

Would you be so kind and point me to either docs or an example you deem high quality?

Reply:
There are a couple of tab characters hiding in the manifest while it's primarily indented with spaces. If the tab display width is set to 8 (which appears to be the current GitHub default), the indentation issues are practically impossible to spot. The manifest needs a tabs-to-spaces treatment :)